### ISSUE TITLE
TPV sister chromatid biorientation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12355]
GO:0031134 sister chromatid biorientation
ancestor to
 GO:0051310 metaphase plate congression
should be removed.

biorientation and plate congression should be siblings. Biorientation is achecved when all pairs of sister kinetochores are attached to microtubules. This is after metaphase plate congression.

see
http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0031134#term=ancchart

Everything else looks OK

**Comment by dosumis** [comment]
https://github.com/geneontology/go-ontology/issues/12244#issuecomment-173562674  has:

mitotic metaphase plate congression
. <- po attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: NEW: lateral attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: sister chromatid biorientation
. <- po NEW: microtubule plus-end directed mitotic chromosome migration

I thought this is what we agreed on that ticket. It's what I implemented.

**Comment by ValWood** [comment]
my fault I misunderstood. It should be

mitotic metaphase plate congression
.  <- po: NEW: lateral attachment of mitotic spindle microtubules to kinetochore
. <- po NEW: microtubule plus-end directed mitotic chromosome migration

attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: NEW: lateral attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: sister chromatid biorientation (this is end-on attachment)

so biorientation  follows congression,  and both have (different parts) which are part of "attachement of spindle microtubules to kinetochore"

**Comment by dosumis** [comment]
Fixed (finally,  I hope...)

**Comment by ValWood** [comment]
HI David, you are going to swear at me. I realised the next day I had made a b***x here but I thought I'd wait until you got back to resurrect.

You were completely correct  before. The arrangement for congression should have been

mitotic metaphase plate congression
. <- po attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: NEW: lateral attachment of mitotic spindle microtubules to kinetochore
. . <- is_a: sister chromatid biorientation
. <- po NEW: microtubule plus-end directed mitotic chromosome migration

I was confused because I thought "microtubule attachment" was not part of "localization" the parent of congression. But now I think it probably is (and if it wasn't  the localization parent of congression could move  down to the localization part).

But the attachment is *definitely* part of congression.

Sorry about that. Very embarrassed to change it twice. I can hear you muttering....
